src/components/keyboard/keyboard.js

```diff
@@ -111,18 +111,18 @@ class KeyboardControl extends BaseComponent {
       if (this.player.muted) {
         this.player.muted = false;
       }
-      if (newVolume <= 1) {
```

Comment:
Just cap newVolume to 1 if it is bigger, no need to add this condition.
You can do it in line 109 where you calculate the newVolume.